Troubleshooting.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting
+
+Here are some of the most common issues you might encounter when using a Skywallet, and the best possible resolutions for troubleshooting these issues. 
+
+If you do not find a resolution for your issue, make sure you contact the support team via - **[store.skycoin.net/pages/support](store.skycoin.net/pages/support)**.
+
+You can also send an email at **support@skycoin.zendesk.com**.
+
+## 1. Lost/Stolen Skywallet
+
+If your Skywallet is lost or stolen, the recommended solution is to **get a new one** as soon as possible. 
+
+Once you got the new Skywallet, then restore the Coins using the Seed of your old Skywallet.
+
+Even though **not a recommended** method, as an emergency step, you can also create a new Desktop Wallet to restore the lost Skywallet.
```

Comment:
`Emergency step` sounds a bit too dramatic imo. It is not recommended, but it is also not a very insecure method. People are using the desktop wallet every day. It is just not as secure.